GroupIS1024/Models/Team.cs

```diff
@@ -10,9 +10,9 @@ public class Team
 
         public string TeamName { get; set; }
 
-        public List<Coach> Coaches { get; set; }
+        public List<Coach>? Coaches { get; set; }
 
-        public List<Player> Players { get; set; }
+        public List<Player>? Players { get; set; }
 
```

Comment:
These changes were suggested in a previously submitted code review.  When that occurs, first in wins.